embedly/tests.py

```diff
@@ -0,0 +1,143 @@
+import json
+import unittest
+import urllib
+
+import mock
+
+import embedly
+
+
+class TestEmbedlyProxy(unittest.TestCase):
+
+    def _get_url_data(self, url):
+        return {
+            'original_url': url,
+        }
+
+    def _get_urls_data(self, urls):
+        return [self._get_url_data(url) for url in urls]
+
+    def _build_query_url(self, urls):
```

Comment:
This should be extracted into a function in embedly.py and used by both the calling request handler and by the test.